changelog/5211.feature.rst

```diff
@@ -0,0 +1,2 @@
+`~sunpy.coordinates.frames.HeliographicStonyhurst` and `~sunpy.coordinates.frames.HeliographicCarrington` now have an ``rsun`` frame attribute to specify the radius of the Sun, which defaults to the value in `sunpy.sun.constants`.
+This frame attribute is used when converting a 2D coordinate to a 3D coordinate.
```

Comment:
Might be worth being clear that `rsun` is not a radial coordinate, but is taken as the default when converting, e.g. something like:

> When the radial coordinate is not explicitly specified, this frame attribute is used as the default radial coordinate when converting a 2D coordinate to a 3D coordinate.

Reply:
Reworder